paper/sections/materials.tex

```diff
@@ -51,13 +83,15 @@ \subsection{Model Flexibility}
 \item the sorting and balancing of samples within the dataset,
 \item the selected SVM hyperparameters, gamma and cost,
 \item the uniformity (or possible lack of) SVM hyperparameters across binary classifiers,
-\item the aggregation technique applied to binary classifiers, and
-\item the prediction method for the final labels.
+\item the aggregation technique applied to binary classifiers,
+\item the prediction method for the final labels, and\\
+\Hamid{for evalueation metrics comment below:}
+{\item the evaluation metrics averaging technique (macro vs micro).}
```

Comment:
@HamidHeyde it looks like you added another bullet here, but didn't actually add any number in the text below which references this bullets. But more importantly, I don't think that this new bullet should actually exist as the "averaging technique" you added IS the "aggregation technique" two bullets above. See the first paragraph of "Performance Evaluation," below. The change on this and the preceding three two lines should be reverted.

Reply:
I just added a bullet point trying to take care of the comment underneath that list. i should have replaced the last bullet point (6) with this one. Because 5 and 6 are kind of the same thing. 👍